contracts/protocol/bases/GroupBase.sol

```diff
@@ -126,7 +126,8 @@ contract GroupBase is ProtocolBase, IBosonGroupEvents {
             valid = (_condition.tokenAddress != address(0) &&
                 _condition.maxCommits > 0 &&
                 _condition.threshold > 0 &&
-                _condition.length == 0);
+                _condition.length == 0 &&
+                _condition.tokenId == 0);
```

Comment:
If token is a multitoken, tokenId 0 is a valid one, so we should not prevent the creation of it.

Reply:
I thought this was the decision we discussed in Slack yesterday. As @mischat said, the ERC1155 per-address condition flow should look like ERC721 and ERC20 per-address condition flow. (any token is accepted)

Reply:
IMO it makes sense since is "per-address" not "per-token"

Reply:
I actually wrote my comment wrong, since this condition is enforcing tokenId to be 0, it's not preventing it.

Anyway, just accepting any token is way too vague. I as a seller might want to allow commits only if the user has a certain amount of ERC1155 tokens, and not just any token.

For example, let's say I have an ERC1155 for a fantasy game. Let's say my tokenIds have the following representations:
0: active user token. Anyone that plays it, has one.
1: common sword
2: rare sword
3: epic sword
4: common armour
5: rare armour

Let's say that I want to offer something only to the users that have at least 1 "epic sword". I'd like to put tokenId = 3 in, but I can't. So if I make a offer, anyone, even if they have only the basic activity token, can commit.
And I cannot go about it, by making a condition with `EvaluationMethod.SpecificToken` and setting it to 3, because I cannot really handle maxCommits there. If I put 1, only 1 person can get it. If I put more (let's say 3), one user can just buy all 3 items. 

The thing that we discussed yesterday, I understood as adding additional support for ranges, but in a way that would not affect the existing conditions. For example, in 2.2.1, if the seller wants to offer something to users that have any type of sword, they need to create 3 offers (one for common, one for rare one for epic). Now with the new support for ranges, a seller could set `tokenId = 1` and `length = 3`, so any user that has one of these three tokens could commit to the offer.

Basically this means, that any combination of `tokenId` and `length` should be ok (and by "any" I mean a combination such that `tokenId+length < maxuint256`